src/core_modules/capture-core/components/WidgetEventEdit/WidgetEventEdit.actions.ts

```diff
@@ -1,11 +1,9 @@
-// @flow
-import type { OrgUnit } from '@dhis2/rules-engine-javascript';
 import { actionCreator } from '../../actions/actions.utils';
 
 export const actionTypes = {
     START_SHOW_EDIT_EVENT_DATA_ENTRY:
         'WidgetEventEdit.StartShowEditEventDataEntry',
 };
 
-export const startShowEditEventDataEntry = (orgUnit: OrgUnit, programCategory: Object) =>
+export const startShowEditEventDataEntry = (orgUnit: any, programCategory: Record<string, unknown>) =>
```

Comment:
Can you rollback `orgUnit: any` and use the type OrgUnit from `import type { OrgUnit } from '@dhis2/rules-engine-javascript';`?